src/containers/MediaElementContainer.js

```diff
@@ -26,21 +27,33 @@ class MediaElementContainer extends Component {
         .getBody();
     } catch (e) {}
 
-    this.prepSources(manifest, choiceItems);
+    this.prepSources(choiceItems);
   }
 
   getSources(choiceItems) {
     const sources = choiceItems.map(item => {
       return {
         src: item.id,
         // TODO: make type more generic, possibly use mime-db
-        type: 'audio/mp4'
+        type: item.getFormat().value
```

Comment:
In a future PR, do you think this key could be changed to `format` to align with the manifest instead of `type` which I confused with the `type` in the manifest?

Reply:
Yeah :+1: